src/TurnService.ts

```diff
@@ -0,0 +1,71 @@
+import Color from './color';
+
+class TurnService{
+
+  public getThreeColor(side: Color[], sideVals: number[]): any{
```

Comment:
```suggestion
  public getColorsByIndexes(side: Color[], sideVals: number[]): Color[] {
```
You should try ur best to avoid "any" when using ts